uikit/views/src/main/res/layout-land/layout_simplified_signpost_view_internal.xml

```diff
@@ -0,0 +1,93 @@
+  ~ in the Software without restriction, including without limitation the rights
+  ~ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+  ~ copies of the Software, and to permit persons to whom the Software is
+  ~ furnished to do so, subject to the following conditions:
+  ~
+  ~ The above copyright notice and this permission notice shall be included in all
+  ~ copies or substantial portions of the Software.
+  ~
+  ~ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  ~ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  ~ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+  ~ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  ~ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  ~ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+  ~ SOFTWARE.
+  -->
+
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <merge
+        tools:background="@drawable/bg_signpost_rounded"
+        tools:ignore="ContentDescription"
+        tools:layout_height="wrap_content"
+        tools:layout_margin="@dimen/signpostViewLayoutMargin"
+        tools:layout_width="344dp"
+        tools:parentTag="androidx.constraintlayout.widget.ConstraintLayout">
+
+        <ImageView
+            android:id="@+id/signpostPrimaryDirectionImageView"
+            android:layout_width="32dp"
+            android:layout_height="32dp"
+            android:layout_marginStart="16dp"
+            app:layout_constraintBottom_toBottomOf="@+id/signpostDistanceTextView"
+            app:layout_constraintEnd_toStartOf="@+id/signpostDistanceTextView"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="@+id/signpostDistanceTextView"
+            tools:src="@drawable/ic_direction_left_45" />
+
+        <TextView
+            android:id="@+id/signpostDistanceTextView"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="8dp"
+            android:layout_marginTop="8dp"
+            android:layout_marginEnd="8dp"
+            android:layout_marginBottom="8dp"
+            android:fontFamily="sans-serif-medium"
+            android:maxLines="1"
+            android:textColor="?attr/signpostTintColor"
+            android:textSize="36sp"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toStartOf="@+id/signpostSecondaryDirectionContainer"
+            app:layout_constraintStart_toEndOf="@id/signpostPrimaryDirectionImageView"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:text="200m"
+            tools:textColor="@color/white" />
+
+        <LinearLayout
```

Comment:
preco tu je linear s 1 view iba?

Reply:
jak už jsem vysvětloval, je to pozůstatek, původně tu byly dva childy tak jak v portrait, ale pak jsem zjistil, že tu být nemusí a že bindig z něj jen udělá nullable, tak jsem ho dal pryč ;)